pyproject.toml

```diff
@@ -101,3 +101,4 @@ exclude = [
     "**/__pycache__",
     "starknet_py/tests/e2e/docs",
 ]
+stubPath = ""
```

Comment:
Why is that needed?

Reply:
The 3rd point in PR description:

- Fix error `"stubPath /.../starknet.py/typings is not a valid directory."` when running typechecker

The error does nothing bad, just annoying to see.

Reply:
Interesting, I never noticed this error but if it fixes it, I guess we can keep it.

Reply:
cc @Solpatium @war-in

Reply:
https://github.com/microsoft/pyright/issues/777

Reply:
It's OK for me but please add a comment about why we need that